experiments/tester/src/RNTester/TestComponents/Callout/CalloutTest.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { ScreenRect, Text, View } from 'react-native';
+import { Button, Callout, Separator } from '@fluentui/react-native';
+import { fabricTesterStyles } from '../Common/styles';
+
+export const CalloutTest: React.FunctionComponent<{}> = () => {
+  const [state, setState] = React.useState({
+    showStandardCallout: false,
+    showCustomizedCallout: false
+  });
+
+  const stdBtnRef = React.useRef<Text>(null);
+
+  const custBtnRef = React.useRef<Text>(null);
+
+  const toggleShowStandardCallout = React.useCallback(() => {
+    setState({ ...state, showStandardCallout: !state.showStandardCallout });
+  }, [state, setState]);
+
+  const toggleShowStandardCalloutInner = React.useCallback(() => {
+    setState({ ...state, showStandardCallout: !state.showStandardCallout });
+  }, [state, setState]);
+
+  const toggleShowCustomizedCallout = React.useCallback(() => {
+    setState({ ...state, showCustomizedCallout: !state.showCustomizedCallout });
+  }, [state, setState]);
+
+  const toggleShowCustomizedCalloutInner = React.useCallback(() => {
+    setState({ ...state, showCustomizedCallout: !state.showCustomizedCallout });
+  }, [state, setState]);
+  const myRect: ScreenRect = { screenX: 10, screenY: 10, width: 100, height: 100 };
+  return (
+    <View>
+      <Text ref={stdBtnRef} style={fabricTesterStyles.testSection}>
+        Standard Usage
+      </Text>
+      <Separator />
+      <Button content="Press for Callout" onClick={toggleShowStandardCallout} />
+
+      {state.showStandardCallout && (
+        <Callout anchorRect={myRect}>
+          <View style={{ height: 200, width: 400 }}>
```

Comment:
Can you set width and height on the Callout directly and avoid the View?

Reply:
Nope; per API review we're having the callout size to contents, and not providing direct w/h props